### ISSUE TITLE
Newbie issue with mlogfilter

### ISSUE THREAD

**Issue by cwood32** [rueckstiess/mtools#329]
Hello, 

Thank you for this tool. I'm new using this tool so this maybe a non issue. I'm using the mlogfilter command trying to find all operations that are listed as query and slower that 1000 ms. When executing I do not get the correct results. I see in my log file that there is a query that is over 1000 ms, but when I use the mlogfilter I don't not see that line. Does mtools cache data or I'm I doing something wrong. This is the script I'm using. I'm using the correct case for the namespace 

python mlogfilter.py c:\mongodb.log --operation query --namespace WickedWorkload_2.Patient --slow 1000

Here is the log entry (edit for space and information). 

Wed Jan 28 00:31:16.302 [conn35757] warning: log line attempted (18k) over max size(10k), printing beginning and end ... getmore WickedWorkload_2.Patient query: { cwlid: ObjectId('54c880cd0ff5e91b040908a5'), pid: { $in: [  ] }, fp: false } cursorid:2301726948581717087 ntoreturn:0 keyUpdates:0 numYields: 51 locks(micros) r:24712 nreturned:1824 reslen:256403 1335ms

**Comment by rueckstiess** [comment]
Hi @cwood32 

There are two problems with this particular line. 

a) the log line is too long, and has the `warning: log line attempted...` string at the beginning. Therefore mtools doesn't recognize it as a typical operation. There is an open request to fix that (and ignore the warning), see #133. I should really fix that soon. 

b) this particular line is actually a "getmore", which is a request for another batch after the initial query. These are tracked as separate operations. So even if the line wasn't too long, it would not show up as query. If you want to include these in your filter you'd have to run something like this:

```
python mlogfilter.py c:\mongodb.log --operation query getmore --namespace WickedWorkload_2.Patient --slow 1000
```

I hope this answers your question, and I will schedule #133 to be included for the next release. 

Thomas

**Comment by cwood32** [comment]
Thank you very much for your answer. That help me out.
